.circleci/config.yml

```diff
@@ -57,14 +57,14 @@ jobs:
       - run:
           command: |
             cd ~/repo/examples/gradle-example
-            ./gradlew clean
+            gradle clean
       # Download and cache dependencies
       - restore_cache:
           keys:
             - v3-dependencies-{{ checksum "pom.xml" }}-{{ checksum "examples/maven-example/pom.xml" }}-{{ checksum "examples/gradle-example/build.gradle" }}
       - run: |
           cd ~/repo/examples/gradle-example
-          ./gradlew build
+          gradle build -x test
```

Comment:
Why do we want to exclude tests? Is there a way to make this step more descriptive (so we know why in the future)?

Reply:
Glad you noticed. 

This is so we don’t run the tests until we want the tests run, in the test phase of the pipeline.

Reply:
I am not seeing the unit tests run at all now locally when I run a maven or gradle build.  This seems strange to me.  Maybe i am doing something wrong?  If it is broken, we should fix it.

Reply:
> Why do we want to exclude tests? Is there a way to make this step more descriptive (so we know why in the future)?

Agree.  This needs to be documented better.

Reply:
I checked on the `gradle -x test` bit. the examples don't have any tests anyway, which I didn't realize, so the change I made is unecessary. I'll revert the change.